build-configs/BuildConfigType.js

```diff
@@ -115,5 +116,6 @@ export type iOSBuildConfigType = {|
   bundleIdentifier: string, // iOS application identifier.
   provisioningProfileSpecifier: string, // Provisioning profile to sign the app.
   appleId: string, // Id of the app in the Apple App Store
+  itunesAppName: string,
```

Comment:
A comment with more information on what this is would be nice I think.

Reply:
done